autode/calculation.py

```diff
@@ -423,16 +424,17 @@ def clean_up(self,
 
         filenames = self.input.filenames
         if everything:
-            filenames.append(self.output.filename)
```

Comment:
Should be left as the output filename doesn't, necessarily, start with the calculation name